music_assistant/server/controllers/player_queues.py

```diff
@@ -890,11 +915,17 @@ async def on_player_register(self, player: Player) -> None:
                     str(err),
                 )
         if queue is None:
+            dont_stop_the_music_enabled = self.mass.config.get_raw_core_config_value(
+                self.domain,
+                CONF_DEFAULT_DONT_STOP_THE_MUSIC,
+                True,
```

Comment:
Why not define the configentry as constant and refer to it here ?
Also.. not so sure about the default set to true.

Reply:
Looked at the default value, but don't see the point in making it a constant. The config is only referred to once, at the time of when the player queue is created. This is the only time we actually need to read the default value from the settings. In all other cases we need the value that is stored on the player_queue. To me it seems similar as for example this setting: https://github.com/music-assistant/server/blob/10fb892d31948f1955881d52b985c26126a47a4d/music_assistant/server/controllers/player_queues.py#L1471

Reply:
then at least make it a constant and set the default to false

Reply:
or let it say otherwise: the default can be true but then we need to ensure there are providers present that have dynamic tracks feature

Reply:
Fixed, Default is now a constant `True`. I also check whether there is a provider available with the similar tracks feature when using the default value.